docs/build.sh

```diff
@@ -3,25 +3,13 @@
 # This was adapted from https://github.com/dgraph-io/dgraph/blob/master/wiki/scripts/build.sh
 #
 
-set -e
```

Comment:
For some reason, this would not work if I left it in. Don't think it's a huge issue

Reply:
> `-e `    When this option is on, if a simple command fails for any of the reasons listed in Consequences of
       Shell  Errors or returns an exit status value >0, and is not part of the compound list following a
       while, until, or if keyword, and is not a part of an AND  or  OR  list,  and  is  not  a  pipeline
       preceded by the ! reserved word, then the shell shall immediately exit.

Reply:
Yes - I understand that - I'm just unsure what the error is because I don't get an output. I'll put it a commit that reinstates it. It may help clarify